src/components/trends/details/graphs/successGraph.jsx

```diff
@@ -40,11 +45,18 @@ const SuccessGraph = ({successCount, failureCount}) => {
         }],
         fill: 'end'
     };
-
+    const opt = _.cloneDeep(options);
```

Comment:
this would create new option object per render, should keep this as const before the render/component function.